Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.15.2
+FROM alpine:latest
```

Comment:
Could you set this to `3.16.1`? My preference is for fixed versions

Reply:
I understand, but the reason I set latest is what I said above: It won't get cached otherwise by Google and one can easily run into limits

Reply:
okay, that makes sense I guess